### PR TITLE
PHX-131/ Text changes 

### PR DESCRIPTION
**Pull request by hasnain-naveed**

### CODE REVIEW

**Comment by chrisndodge** [comment]
@catong do you want to review the text changes?

**Comment by chrisndodge** [comment]
@griffresch I see you want to change 3 links at the bottom to just one. What URL is that supposed to point to? Is it an existing URL or one that needs to be created?

@hasnain-naveed I'm asking this because if we have a different URL then we'll have to update configuration via DevOps.

**Comment by chrisndodge** [comment]
@griffresch @hasnain-naveed OK, I just answered my own question. I'll comment in the code...

File: edx_proctoring/templates/proctoring/seq_proctored_practice_exam_footer.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<div class="faq-proctoring-exam">
+    <p>
+      <a class="footer-link" href="{{link_urls.about_proctored_exams}}" target="_blank">
```
Comment (chrisndodge) [inline edx_proctoring/templates/proctoring/seq_proctored_practice_exam_footer.html]: please use the link_urls.faq here that configuration has the URL that we need. Thanks

File: settings.py [hunk 2]
```diff
@@ -88,6 +88,7 @@
         'faq': '',
         'contact_us': '',
         'tech_requirements': '',
+        'about_proctored_exams': ''
```
Comment (chrisndodge) [inline settings.py]: Drop this one as we don't need another URL

File: edx_proctoring/templates/proctoring/seq_proctored_exam_footer.html [hunk 3]
```diff
@@ -1,21 +1,10 @@
 {% load i18n %}
 <div class="faq-proctoring-exam">
-  <h4> {% trans "See Also" %} </h4>
     <p>
-      <a class="footer-link" href="{{link_urls.faq}}" target="_blank">
+      <a class="footer-link" href="{{link_urls.about_proctored_exams}}" target="_blank">
```
Comment (chrisndodge) [inline edx_proctoring/templates/proctoring/seq_proctored_exam_footer.html]: please use link_urls.faq here as that has the correct configuration right now

**Comment by chrisndodge** [comment]
@catong sorry, I should note that a review is due by end-of-day Thursday as we have to make these changes into the Release Candidate that will be cut on Friday.

File: edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html [hunk 4]
```diff
@@ -50,15 +38,15 @@
...
+        If you take this exam without proctoring, you will <strong> no longer be eligible for academic credit. </strong>
       {% endblocktrans %}
     </div>
     <div class="proctored-exam-skip-actions">
-      <button class="proctored-exam-skip-cancel-button btn btn-default btn-base">
-        {% trans "Cancel" %}
-      </button>
       <button class="proctored-exam-skip-confirm-button btn btn-primary btn-base" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}">
-        {% trans "Yes, Take this as an open exam" %}
+        {% trans "Continue Exam without Proctoring" %}
```
Comment (catong) [inline edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html]: Make consistent initial caps on all words, including "without"

**Comment by catong** [comment]
@chrisndodge Thanks, I've taken a quick pass through the text changes, and it looks fine, though I can't visualize how some of those headings look without punctuation. 
The one remaining URL for "About Proctored Exams" should go to this URL -- it should have been one of the existing ones.
http://edx.readthedocs.org/projects/edx-guide-for-students/en/latest/SFD_credit_courses/SFD_proctored_exams.html

**Comment by chrisndodge** [comment]
Thanks @catong. I should have the sandbox refreshed tomorrow (9/10)

**Comment by chrisndodge** [comment]
@hasnain-naveed moving PR to be against the new release branch (rc/2015-09-16)

**Pull request closed without merging**